servers/src/main/java/tachyon/worker/block/BlockWorker.java

```diff
@@ -96,6 +97,8 @@
   private final UIWebServer mWebServer;
   /** Worker metrics system */
   private MetricsSystem mWorkerMetricsSystem;
+  /** Lineage worker */
+  private LineageWorker mLineageWorker;
```

Comment:
Does it seem strange that the BlockWorker has a LineageWorker? Maybe a TachyonWorker should have multiple workers like BlockWorker, LineageWorker, and so on? What do you think?

If this should be restructured, it would be a different PR.